lms/djangoapps/grades/models.py

```diff
@@ -376,6 +423,14 @@ def _prepare_params_visible_blocks_id(cls, params):
         params['visible_blocks_id'] = params['visible_blocks'].hash_value
         del params['visible_blocks']
 
+    def remove_attempts(self):
```

Comment:
Yeah, I was thinking it might be useful for something like reset_student_attempts (or maybe clear_student_state).  @nasthagiri Opinions?